### PR TITLE
update content

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
removed bullet point and added heading from http://www.vets.gov/education/gi-bill/post-9-11/ch-33-benefit/

## Testing done
ran locally

## Testing Plan
none

## Screenshots
download-2

## Acceptance Criteria (Definition of Done)
copy changes verified

#### Applies to all PRs

- [x] Provide link to originating GitHub issue, or connected to it via ZenHub

### CODE REVIEW

**Review by cehsu — APPROVED** [review]
LGTM, pending @peggygannon's approval.

**Review by erikphansen — APPROVED** [review]

**Review by peggygannon — APPROVED** [review]
@rianfowler Looks good.

**Pull request merged**
